src/thor/astar_bss.cc

```diff
@@ -156,7 +156,9 @@ void AStarBSSAlgorithm::ExpandForward(GraphReader& graphreader,
 
     auto edge_cost = current_costing->EdgeCost(directededge, tile);
     Cost normalized_edge_cost = {edge_cost.cost * current_costing->GetModeFactor(), edge_cost.secs};
-    auto transition_cost = current_costing->TransitionCost(directededge, nodeinfo, pred);
+    auto reader_getter = [&]() { return baldr::LimitedGraphReader(graphreader); };
```

Comment:
can we limit the scope of capture to just the reader?
```suggestion
    auto reader_getter = [&graphreader]() { return baldr::LimitedGraphReader(graphreader); };
```